tests/lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -44,6 +44,40 @@ eslintTester.run('avoid-leaking-state-in-components', rule, {
       code: 'export default Component.extend({ classNames: [], classNameBindings: [], actions: {}, concatenatedProperties: [], mergedProperties: [], positionalParams: [] });',
       parserOptions: { ecmaVersion: 6, sourceType: 'module' },
     },
+
+    {
```

Comment:
I don't think it's necessary to duplicate the tests. You can just change existing tests to be `Foo.extend` instead of `Component.extend`.